test/test_utils.py

```diff
@@ -245,11 +246,12 @@ def test_draw_segmentation_masks(colors, alpha):
         torch.testing.assert_close(out[:, mask], interpolated_color, rtol=0.0, atol=1.0)
 
 
-def test_draw_segmentation_masks_errors():
+@pytest.mark.parametrize("device", cpu_and_cuda())
+def test_draw_segmentation_masks_errors(device):
```

Comment:
I'll leave this here to avoid churn but we typically don't need to run those tests on cuda, as there's nothing cuda-spcecific there. Same for `test_draw_no_segmention_mask`